src/migrator.coffee

```diff
@@ -29,9 +33,12 @@ validateOptions = (optionsToCheck) ->
   if optionsToCheck["on-read-error"]? && ["ignore", "log", "exit"].indexOf(optionsToCheck["on-read-error"]) == -1
     throw "invalid value for 'on-read-error': '#{optionsToCheck["on-read-error"]}'"
 
-  if optionsToCheck["command-buffering"]? && !isPositiveInteger(optionsToCheck["command-buffering"])
+  if optionsToCheck["command-buffering"]? && !validateNumber(optionsToCheck["command-buffering"], min: 1, integer: true)
```

Comment:
may want to be explicit here with `{min: 1, integer: true}` or bring it to a new line

```
!validateNumber(optionsToCheck["command-buffering"], 
  min: 1
  integer: true)
```